api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/BaseDataSet.java

```diff
@@ -54,6 +56,15 @@ public void addRow(
     return parameters;
   }
 
+  public List<Parameter> getDataQualityParameters() {
```

Comment:
Am calling this method in 16 datasets for this report, and I though I could place it in a convenient place where I could just call it without duplicating in every dataset

Reply:
better to create a parent data set class for these datasets then have all of them inherit from it since they all share this.

This only applies to data quality data sets so better we isolate it to them only or find a way to extend the existing logic to allow us to add more parameters.

Reply:
fixed